cloud/service-mapping/ba-management.md

```diff
@@ -215,8 +214,7 @@ In Centreon Cloud, when you create a business activity, you must always add it t
 
 #### Notification
 
-Configure when and for what status notifications should be sent when the BA's
-status changes. Add the **Notification** contact group so that users can be notified.
+This feature is not currently active.
```

Comment:
mettre un commentaire du genre 
"BAs use common notification mechanism see [here ](https://docs.centreon.com/cloud/alerts-notifications/notif-configuration/)for more details